### DISCUSSION TITLE
Tags for accessibility

### DISCUSSION THREAD

**Discussion by aslenko** [LibrePDF/OpenPDF#782] [Q&A]
Where can I find examples of how to apply tags similar to this https://helpx.adobe.com/acrobat/using/creating-accessible-pdfs.html?

**Comment by asturio** [comment]
I really don't know if there is support for that in OpenPDF, and if so, how well it is supported. There is a `TagMap`-class in OpenPDF. Is that what you need?

Did you try anything? Any success? What is needed to obtain that?

Pull requests are welcome.

**Reply by arnthom** [reply]
Is there any progress in creating acessible pdf with OpenPDF?
